addon/services/router-scroll.js

```diff
@@ -54,19 +71,26 @@ export default Service.extend({
     set(this, 'key', stateUuid); // eslint-disable-line ember/no-side-effects
     const key = getWithDefault(this, 'key', '-1');
 
-    return getWithDefault(scrollMap, key, { x: 0, y: 0 });
+    return getWithDefault(scrollMap, key, scrollMap.default);
   }).volatile(),
 
   _loadConfig() {
     const config = getOwner(this).resolveRegistration('config:environment');
 
-    if (config && config.routerScroll && config.routerScroll.scrollElement) {
+    if (config && config.routerScroll) {
       const scrollElement = config.routerScroll.scrollElement;
+      const targetElement = config.routerScroll.targetElement;
+
+      assert('You defined both scrollElement and targetElement in your config. We currently only support definining one of them', !(scrollElement && targetElement));
 
       if ('string' === typeOf(scrollElement)) {
         set(this, 'scrollElement', scrollElement);
```

Comment:
Should we throw an error if you pass something other than a string?

Reply:
I think since this is coming from the config, we could prob leave it as is (not possible to pass an element or something like that).  What is your opinion on this.  I can easily add it if you would like.